workflow/ush/detect_machine.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 # First detect w/ hostname
 export MACHINE
+
```

Comment:
Suggest removing the extra empty line to make it intact since this PR actually does not change this script.

Reply:
Done